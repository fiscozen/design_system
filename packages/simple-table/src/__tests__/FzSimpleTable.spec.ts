import { describe, it, vi } from "vitest";
import { mount } from "@vue/test-utils";
import { FzSimpleTable } from "..";

describe.concurrent("FzSimpleTable", () => {
  it("matches snaphost", async ({ expect }) => {
    const wrapper = mount(FzSimpleTable, {
      props: {},
      slots: {},
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
