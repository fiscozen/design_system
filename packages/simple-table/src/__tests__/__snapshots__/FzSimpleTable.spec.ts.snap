// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzSimpleTable > matches snaphost 1`] = `
"<table class="w-full text-left rounded overflow-hidden bg-core-white">
  <thead>
    <tr class="bg-grey-100">
      <th class="px-16 h-48 font-medium">User</th>
      <th class="px-16 h-48 font-medium">Date</th>
    </tr>
  </thead>
  <tbody>
    <tr class="border-b-1 border-grey-100">
      <td class="px-16 h-48 text-grey-500">John Doe</td>
      <td class="px-16 h-48 text-grey-500">12/12/2022</td>
    </tr>
    <tr class="border-b-1 border-grey-100">
      <td class="px-16 h-48 text-grey-500">John Doe1</td>
      <td class="px-16 h-48 text-grey-500">10/05/2024</td>
    </tr>
    <!--v-if-->
  </tbody>
</table>"
`;

exports[`FzSimpleTable > matches snaphost empty table 1`] = `
"<table class="w-full text-left rounded overflow-hidden bg-core-white">
  <thead>
    <tr class="bg-grey-100">
      <th class="px-16 h-48 font-medium">User</th>
      <th class="px-16 h-48 font-medium">Date</th>
    </tr>
  </thead>
  <tbody>
    <tr class="text-center text-grey-500">
      <td colspan="100%" class="h-80">No data available</td>
    </tr>
  </tbody>
</table>"
`;
